queries.sql

```diff
@@ -8,3 +8,46 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name <> 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals
+SET species = 'Unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+UPDATE animals
+SET species = 'Digimon' WHERE name LIKE '%mon';
+UPDATE animals
+SET species = 'Pokemon' WHERE species IS NULL OR species = '';
+SELECT * FROM animals;
+COMMIT;
+SELECT * FROM animals;
+
+BEGIN;
+DELETE FROM animals;
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SAVEPOINT my_savepoint;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO my_savepoint;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
```

Comment:
Nice job updating the required queries in the right files😉. I also noticed you added a screenshot of your queries, but there are some issues there. The requirements want you to show screenshot evidence for every query that is run. and you haven't done that for a couple of them. Kindly take a look at each step of your queries and ensure the results are presented in a screenshot and uploaded in the PR.

To make this easier for you, write each query in the PR description and then, under each query show the screenshot result for that specific query. this makes it easy to follow through and understand.😉.

<img width="1021" alt="image" src="https://github.com/SAZZAD1Q2/Vet-Clinic/assets/64643995/1206e8f1-7b3d-4381-a20e-3a4eb89d3fb9">